.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea
+out
```

Comment:
I remember we intentionally removed this from .gitignore in the fork because the imports didn't work otherwise